core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/internal/signer/V4RequestSigningResult.java

```diff
@@ -24,17 +24,15 @@
 @SdkInternalApi
 // TODO(sra-identity-auth): This is currently not @Immutable because signedRequest is a Builder. Is Builder needed? If it could
 //  hold reference to SdkHttpRequest instead, this class would be @Immutable.
-// TODO(sra-identity-auth): Consider if we can rename this to convey something more. maybe,
-//  V4RequestSigningResult/V4RequestSigningResultData? Note there is V4aContext similarly.
-public final class V4Context {
+public final class V4RequestSigningResult {
     private final String contentHash;
     private final byte[] signingKey;
     private final String signature;
     private final V4CanonicalRequest canonicalRequest;
     private final SdkHttpRequest.Builder signedRequest;
 
-    public V4Context(String contentHash, byte[] signingKey, String signature,
-                     V4CanonicalRequest canonicalRequest, SdkHttpRequest.Builder signedRequest) {
+    public V4RequestSigningResult(String contentHash, byte[] signingKey, String signature,
```

Comment:
I'm not getting why V4A has `RequestSigningResult` and V4 has `V4RequestSigningResult`

Reply:
thanks for catching that. I missed a step. I renamed to RequestSigningResult first, and meant to rename to V4aRequestSigningResult but forgot. will update.